### DISCUSSION TITLE
Inclusion of metabolite compartments in reaction equations

### DISCUSSION THREAD

**Discussion by JonathanRob** [MetabolicAtlas/MetabolicAtlas#679] [General]
I noticed that one of the recent updates to Metabolic Atlas involved the removal of metabolite compartment abbreviations (e.g., `[c]`, `[m]`, etc.) from reaction equations shown in the GEM browser. What was the reason behind this change? It was pretty convenient to have these, in particular for reactions involving more than one compartment.

Taking a simple example, the reactions for glucose transport between different compartments all look the same ([MAR05027](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR05027), [MAR07675](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR07675), [MAR05029](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR05029), etc.)

**Comment by mihai-sysbio** [comment]
To be honest, I have noticed this myself but did not report it, hoping it was just me.
If I remember correctly, this was one of the compromises when switching to the Neo4j database. That is not a valid reason, so it would be good to reference this in a new issue.

**Reply by mihai-sysbio** [reply]
This issue has been resolved in the latest release.
